src/clap_app.rs

```diff
@@ -283,10 +283,24 @@ pub async fn init_cli() {
                                     get_pipeline_vars(&mut cm_client, program_id, &pipeline_id)
                                         .await
                                         .unwrap();
+
                                 println!(
                                     "{}",
                                     serde_json::to_string_pretty(&pipeline_vars).unwrap()
                                 );
+                                if let Some(vf) = pipeline_vars
+                                    .variables
+                                    .iter()
+                                    .find(|vf| vf.service == PipelineVariableServiceType::Invalid)
+                                {
+                                    eprintln!(
+                                        "{:>8} {}  '{}: {}'",
+                                        "⚠".yellow(),
+                                        "WARN, invalid service type detected for variable".yellow(),
+                                        vf.name,
+                                        vf.service
+                                    );
+                                }
```

Comment:
Nice @BerSomBen I think it would make sense to also implement the same for environment variables. Can you take over this or should I?

Reply:
of course! I'll moved the block in front of the loop there as well, but did not change the color and sign. 

I'll update it